cedar-policy/CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Cedar Language Version: 4.0
 
 ### Changed
 
+- `EntityUID::from_json` now returns a concrete type instead of an existential type
+- `CedarSchemaError::Parse` is more transparent
```

Comment:
I'd leave these out as covered by the bullet "Significantly reworked". We haven't called out all the other detailed error-type changes